test/functional/interface_nng.py

```diff
@@ -0,0 +1,688 @@
+#!/usr/bin/env python3
```

Comment:
I think the naming of this will trigger a testing error? There's some prefix checks.

Reply:
No, this obeys the naming rules.